geode-redis/src/integrationTest/java/org/apache/geode/redis/general/ShutdownIntegrationTest.java

```diff
@@ -56,6 +57,7 @@ public void shutdownActuallyShutsDownTheServer() {
   }
 
   @Test
+  @Ignore("this test is flaky")
```

Comment:
Should we create a Jira for this?

Reply:
I think I figured out why this test was still flaky. I removed this ignore and added a fix for the flakiness in: https://github.com/apache/geode/pull/5204/commits/823cf5a5c09002e4e543a48ac3a0c7b70232dcc9

Reply:
Good find!